### DISCUSSION TITLE
blog/westlake-interview

### DISCUSSION THREAD

**Discussion by giscus** [wjwei-handsome/wjwei-docusaurus#11] [General]
# blog/westlake-interview

分享一下 2024 年西湖大学生命科学学院博士申请上岸历程和经验

https://www.wjwei.blog/blog/westlake-interview

**Comment by MushroomLyn** [comment]
大佬，能分享一下个人陈述研究计划部分的经验嘛？研究计划是需要给出大致的研究方向，还是要细致到方法呢？我正在写个人陈述，但研究计划部分不知道该怎么进行下去了。QAQ

**Reply by wjwei-handsome** [reply]
你好，感谢关注～ 可以发送邮件给我：
weiwenjie@westlake.edu.cn
或者
wjwei9908@gmail.com

**Comment by XufChen-Vitality** [comment]
看完了感觉 身临其境的紧张了。回想起当初临近毕业做抉择的时候也是一样的茫然，犹豫不决。经历了长时间的挣扎抉择，最终的选择也是无怨无悔。替你开心！
